src/styles/stars.scss

```
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center center;
+  }
+
+  &--1 &__star:nth-child(1),
+  &--2 &__star:nth-child(-n+2),
+  &--3 &__star:nth-child(-n+3),
+  &--4 &__star:nth-child(-n+4),
+  &--5 &__star:nth-child(n) {
```

```suggestion
  &--1 &__star:nth-child(-n+1),
  &--2 &__star:nth-child(-n+2),
  &--3 &__star:nth-child(-n+3),
  &--4 &__star:nth-child(-n+4),
  &--5 &__star:nth-child(-n+5) {
```

Replace it to make all code the same style.